jupyter_archive/tests/test_archive_handler.py

```diff
@@ -144,7 +146,7 @@ async def test_extract(jp_fetch, jp_root_dir, file_name, format, mode):
 
     # Make an archive
     archive_dir_path = jp_root_dir / file_name
-    archive_path = archive_dir_path.with_suffix("." + format)
+    archive_path = archive_dir_path.with_suffix(f"{archive_dir_path.suffix}.{format}")
```

Comment:
I've fixed this line because previous version overwrite extension.

Reply:
Don't you mean changing the line 145 in handlers.py instead?

I think we can simplify that by simply appending the format:

```suggestion
    archive_path = archive_dir_path.parent / f"{archive_dir_path.name}.{format}"
```

Reply:
Thank you for your suggestion! I'll use your code because it is more simplify!

> Don't you mean changing the line 145 in handlers.py instead?

I see, do we need to change handlers.py#L145 for keeping the directory name including `.`?

Reply:
Yes it will be more consistent.